check_out/bill.php

```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Checkout Bill</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/css/bootstrap.min.css">
+</head>
+
+<body>
+
+    <div class="container">
+        <h2 class="my-4">Checkout Bill</h2>
+        <div class="row">
+            <div class="col-md-6">
+                <?php
+                session_start();
+                $sql = "SELECT * FROM user where user_id =1;";
```

Comment:
I don't understand why the user_id must equal one

Reply:
Actually I am trying data so user_id will be 1